schema/formats.go

```diff
@@ -51,6 +55,41 @@ func (f cidrOrIPv4FormatChecker) IsFormat(input string) bool {
 	return (cidrErr == nil && cidrIP.To4() != nil) || (ip != nil && ip.To4() != nil)
 }
 
+func (f ipv4NetworkFormatChecker) IsFormat(input string) bool {
+	hostIP, netIP, cidrErr := extractHostAndNet(input)
+	if cidrErr != nil {
+		return false
+	}
+	return reflect.DeepEqual(hostIP, netIP)
+}
+
+func (f ipv4AddressWithCidrFormatChecker) IsFormat(input string) bool {
+	hostIP, netIP, cidrErr := extractHostAndNet(input)
+	if cidrErr != nil {
+		return false
+	}
+
+	for i := range hostIP {
+		if hostIP[i] == 255 {
```

Comment:
this is not going to work for cases where prefix length isn't multiple of 8, e.g. Broadcast address for subnet 192.168.2.0/25 is 192.168.2.127.

Reply:
true, fixed

Reply:
This is still not fixed, please check https://play.golang.org/p/46MHiZHeuMs for correct method of computing broadcast address.